backend/src/Controllers/getBallotsByElectionIDController.ts

```diff
@@ -0,0 +1,30 @@
+import ServiceLocator from "../ServiceLocator";
+import Logger from "../Services/Logging/Logger";
+import { BadRequest } from "@curveball/http-errors";
+import { Ballot } from '../../../domain_model/Ballot';
+import { Score } from '../../../domain_model/Score';
+import { expectPermission } from "./controllerUtils";
+import { permissions } from '../../../domain_model/permissions';
+
+const BallotModel = ServiceLocator.ballotsDb();
+
+const getBallotsByElectionID = async (req: any, res: any, next: any) => {
+    var electionId = req.election.election_id;
+    Logger.debug(req, "getBallotsByElectionID: " + electionId);
+
+    expectPermission(req.user_auth.roles, permissions.canViewBallots)
+
+    const ballots = await BallotModel.getBallotsByElectionID(String(electionId), req);
+    if (!ballots) {
+        const msg = `Ballots not found for Election ${electionId}`;
+        Logger.info(req, msg);
+        throw new BadRequest(msg)
+    }
+    Logger.debug(req, "ballots = ", ballots);
+    req.ballots = ballots;
```

Comment:
can we remove this?  (just put the ballots in the res.json)

Reply:
Good catch, left over from old code. Updated in recent commit